### PR TITLE
DO NOT MERGE - THIS IS TO SEE CI RESULT - Rebase/april 8

### PR DESCRIPTION
**Pull request by echarles**
DO NOT MERGE - THIS IS TO SEE CI RESULT - Rebase/april 8 

Let's see what this gives

### CODE REVIEW

**Comment by jtpio** [comment]
> DO NOT MERGE - THIS IS TO SEE CI RESULT

-> converting to draft

**Comment by jtpio** [comment]
The failing `splice_source` check (new from the rebase) might also be related to the new packages not being available on npm yet:

https://github.com/jupyter-rtc/jupyterlab/pull/46/checks?check_run_id=2295634136

```
error Couldn't find package "@jupyterlab/nbmodel@^3.1.0-alpha.3" required by "@jupyterlab/cells@/home/runner/work/jupyterlab/jupyterlab/packages/cells" on the "npm" registry.
```

**Comment by echarles** [comment]
`js-notebook` is still failing.

File: packages/codemirror/style/base.css [hunk 1]
```diff
@@ -226,3 +226,39 @@
 .cm-s-jupyter .CodeMirror-gutter {
   background-color: var(--jp-layout-color2);
 }
+
+/* Styles for shared cursors (remote cursor locations and selected ranges) */
+.remote-caret {
```
Comment (jtpio) [inline packages/codemirror/style/base.css:231]: We should double check whether this rebase picked up all the changes.

For example there was a quick pass on the CSS in https://github.com/jupyter-rtc/jupyterlab/pull/41/, which resulted in prefixing these lines:

https://github.com/jupyter-rtc/jupyterlab/blob/82c40544f464b455b1c6060e80fcb53a8321355b/packages/codemirror/style/base.css#L231

But here in this diff they don't seem be taken into account.

**Pull request closed without merging**